Vulnerabilities/securitymisconfiguration.md

```diff
@@ -0,0 +1,27 @@
+# [securitymisconfiguration](https://owasp.org/www-project-top-ten/2017/A6_2017-Security_Misconfiguration)
```

Comment:
`securitymisconfiguration` => `Security Misconfiguration` ? (seems consistent with what we have elsewhere)

Reply:
Lol good catch